src/main/java/com/feed_the_beast/ftbu/cmd/CmdChunks.java

```diff
@@ -52,15 +53,23 @@ public void execute(MinecraftServer server, ICommandSender sender, String[] args
             IForgePlayer p = FTBLibIntegration.API.getForgePlayer(player);
             ChunkDimPos pos = new EntityDimPos(player).toBlockDimPos().toChunkPos();
 
-            if(FTBUUniverseData.claimChunk(p, pos))
-            {
-                FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CHUNK_CLAIMED);
-                updateChunk(player, pos);
+
+            if(FTBUConfigWorld.CHUNK_CLAIMING.getBoolean()) {
+                if(FTBUUniverseData.claimChunk(p, pos))
+                {
+                    FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CHUNK_CLAIMED);
+                    updateChunk(player, pos);
+                }
+                else
+                {
+                    FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CANT_MODIFY_CHUNK);
+                }
             }
             else
             {
-                FTBLibIntegration.API.sendNotification(player, FTBUNotifications.CANT_MODIFY_CHUNK);
+                sender.addChatMessage(new TextComponentString("Error! Claiming is not enabled on this server."));
```

Comment:
I prefer to have it as a Notification in FTBUNotifications:

@RegistryObject
public static final INotification CANT_MODIFY_CHUNK = create("cant_modify_chunk", 1).setError(new TextComponentString("Claiming is not enabled on this server!"));